environments/el-9/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN set -x && \
     dnf install -y --enablerepo epel --enablerepo devel  \
         findutils gcc gcc-c++ make patch bzip2 curl autoconf automake \
         openssl-devel libyaml-devel libffi-devel readline-devel zlib-devel \
-        gdbm-devel ncurses-devel && \
+        gdbm-devel ncurses-devel \
+        rust-toolchain && \
```

Comment:
centos-8 and all other Docker images now build properly. But this here (for el-9) needs to be changed to rust-toolset too.